src/arrays.js

```diff
@@ -1,61 +1,72 @@
 const getNthElement = (index, array) => {
-  // your code here
+if (index > array.length -1) {
+  return array[index % array.length]
+} else { return array[index]}
```

Comment:
One thing you might want to look at here which is pretty cool, is the [ternary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator). You can use it in scenarios like this one to cut down on if/else statements:

```js
return (index > (array.length - 1)) 
  ? array[index % array.length] 
  : array[index]
```

Please do ensure you're always indenting 2 spaces inside of a block `{}` of code.

Note also how I've grouped the addition within your if statement - this also makes the code easier to read.